Source/FullScreenPokemon.ts

```diff
@@ -4888,6 +4888,52 @@ module FullScreenPokemon {
             player.FSP.killNormal(player.bordering[player.direction]);
         }
 
+        /**
+         * Makes a StrengthBoulder move.
+         *
+         * @param player   The Player.
+         * @param pokemon   The Pokemon using Strength.
+         * @todo Eventually add check to make sure the Player beat the Gym leader needed to use the move.
+         * @todo Add an animation for what happens when the StrengthBoulder is moved.
```

Comment:
Also a `@todo` to replace the two `registerB` calls with a `closeAllMenus` call.